weave/integrations/integration_utilities.py

```diff
@@ -92,6 +92,11 @@ def op_name_from_ref(ref: str) -> str:
     return ref.split("/")[-1].split(":")[0]
 
 
+def op_name_from_call(call: Call) -> str:
+    op_name = cast(str, call._op_name)
```

Comment:
Unsafe cast. could be a future. Either use `op_name` which will wait on the future or do a type check first